tools/sgcollect.py

```diff
@@ -178,12 +185,15 @@ def create_option_parser():
     parser.add_option(
         "--sync-gateway-username",
         dest="sync_gateway_username",
-        help="Sync Gateway Admin API username ",
+        help="Sync Gateway Admin API username. Can be specified with SG_USERNAME env variable.",
     )
     parser.add_option(
         "--sync-gateway-password",
-        dest="sync_gateway_password",
-        help="Sync Gateway Admin API password ",
+        action="callback",
+        callback=sync_gateway_option_callback,
+        dest="do_not_use",
+        type=str,
+        help="Sync Gateway Admin API password. Can be specified with SG_PASSWORD env variable.",
```

Comment:
Since any use of this flag will error now, steer people away from it.

```suggestion
        help="Sync Gateway Admin API password. Should be specified with SG_PASSWORD env variable or input interactively.",
```

Reply:
@torcolvin Do we want this change or leave it?